source/concepts/faq.md

```diff
@@ -60,6 +60,15 @@ There are downsides to relying on [experimental features](https://nixos.org/manu
 - The [Nix documentation team](https://nixos.org/community/teams/documentation.html) focuses on improving documentation and learning materials for stable features and common principles.
   When using flakes, you will have to rely more heavily on user-to-user support, third-party documentation, and the source code.
 
+## Which channel branch should I use?
+
+If you are on NixOS, use a channel branch with the prefix `nixos-*`.
+If you are on another OS, use `nixpkgs-*`.
+The former is built for NixOS test acceptance, e.g., booting NixOS; while the latter, for non-NixOS test acceptance, e.g., running a Linux builder on macOS.
+Make sure `*` matches your desired configuration for release, architecture and/or channel type.
```

Comment:
I makes more sense to group information that belongs together: it's easier to read (don't have to wire it up in your head) and easier to maintain (can shuffle around the whole unit instead of picking apart sentences).

```suggestion
- On Linux (including NixOS and WSL), use a channel branch with the prefix [`nixos-*`](https://github.com/NixOS/nixpkgs/branches/all?query=nixos-).

  These branches only contain commits that passed the NixOS test suite.
  
- On any other platform, use [`nixpkgs-*`](https://github.com/NixOS/nixpkgs/branches/all?query=nixpkgs-).

  These branches contain commits that passed the test suites specific to other platforms.
   
Each channel branch follows the corresponding [`release-*`](https://github.com/NixOS/nixpkgs/branches/all?query=release-) branch.
For example, `nixos-23.05` and `nixpkgs-23.05-darwin` are both based on `release-23.05`.
 
`*-unstable` follows `master`.
 
`*-small` channel branches have passed a smaller test suite, which means they are more up-to-date with respect to their base branch, but have fewer binaries available in the [public cache](http://cache.nixos.org/).
```

@raitobezarius is that technically and morally correct? We need most of that information (and a lot more detail) in Nixpkgs contributor docs, so we can referecence that from here. But adding it here to have a quick answer.

Reply:
It's not great, but I think these are the current conventions:
- Stable (follows release-*)
  - Darwin: nixpkgs-*-darwin
  - Else: nixos-*
- Unstable (follows master)
  - NixOS: nixos-unstable
  - Else: nixpkgs-unstable

Reply:
Smaller channels don't have "less" binaries per se, this is a less-tested channel, e.g. subject to more breakages like Firefox not running or what not.

(reviewing.)

Reply:
```suggestion
Nixpkgs and NixOS have both stable and rolling releases.

### Stable

- On Linux (including NixOS and WSL), use [`nixos-*`](https://github.com/NixOS/nixpkgs/branches/all?query=nixos-).

  These branches only contain commits that passed the NixOS test suite.
- On any other platform, use [`nixpkgs-*`](https://github.com/NixOS/nixpkgs/branches/all?query=nixpkgs-).

  These branches contain commits that passed the test suites specific to other platforms.

All of these "channel branches" follow the corresponding [`release-*`](https://github.com/NixOS/nixpkgs/branches/all?query=release-) branch.

:::{example}
`nixos-23.05` and `nixpkgs-23.05-darwin` are both based on `release-23.05`.
:::

### Rolling

- On Linux (including NixOS and WSL), use [`nixos-unstable`](https://github.com/NixOS/nixpkgs/branches/all?query=nixos-unstable).
- On any other platform, use [`nixpkgs-unstable`](https://github.com/NixOS/nixpkgs/branches/all?query=nixpkgs-unstable).

These branches follow `master`, the main development development branch.

[`*-small`](https://github.com/NixOS/nixpkgs/branches/all?query=-small) channel branches have passed a smaller test suite, which means they are more up-to-date with respect to their base branch but offer fewer stability guarantees.
```